test/e2e/util.go

```diff
@@ -90,3 +93,62 @@ func patchAndCheckRoute(t *testing.T, client *testframework.Clientset, isOperato
 	})
 	return err
 }
+
+func mergeAndCheckConfigMap(t *testing.T, client *testframework.Clientset, isOperatorManaged bool) error {
+	t.Logf("merging Data on the console ConfigMap")
+	expectedBranding := "branding: online"
+
+	cm := &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "console-config",
+			Namespace: "openshift-config-managed",
+		},
+		Data: map[string]string{
+			"console-config.yaml": `kind: ConsoleConfig
+apiVersion: console.openshift.io/v1
+customization:
+  ` + expectedBranding + `
+  documentationBaseURL: https://docs.okd.io/4.0/
+`,
+		},
+	}
+
+	originalConfigMap, err := testframework.GetConsoleConfigMap(client)
+	if err != nil {
+		return err
+	}
+
+	_, err = client.ConfigMaps(consoleapi.OpenshiftConfigManagedNamespace).Update(cm)
+	if err != nil {
+		return err
+	}
+
+	t.Logf("polling for new branding value on the console ConfigMap")
+	err = wait.Poll(5*time.Second, pollTimeout, func() (stop bool, err error) {
+		configMap, err := testframework.GetConsoleConfigMap(client)
+		if err != nil {
+			return true, err
+		}
+		brandingValue := getBrandingValue(originalConfigMap)
+		brandingValue = getBrandingValue(configMap)
```

Comment:
Not sure whats the point of initialising  the variable and then overwriting it

Reply:
Bad cut and paste.